spec/rubocop/config_spec.rb

```diff
@@ -526,6 +544,45 @@
         expect(configuration.cop_enabled?(cop_class)).to be true
       end
     end
+
+    context 'when DisabledByDefault is true' do
+      let(:hash) do
+        {
+          'AllCops' => { 'DisabledByDefault' => true },
+          'Style/Copyright' => { 'Exclude' => ['foo'] }
```

Comment:
Nope.  According to the docs, anything listed in the user config is enabled.

That's not the failing test.  The failing one is the one that's not disabling a cop that's enabled by default, even though DisabledByDefault is true.